tests/search_test.py

```diff
@@ -81,18 +81,24 @@ def test_should_add_with_field_as_dict(self):
         self.assertEqual(query, {"with_field": ["context", "tags"]})
 
     @unittest.skipUnless(cloudinary.config().api_secret, "requires api_key/api_secret")
+    @unittest.skip("For this test to work, 'Advanced search' should be enabled for your cloud. " +
```

Comment:
These tests need to work in TravisCI. Thus this needs to be enabled by configuration (e.g. environment variable).